### PR TITLE
remove error control operators

### PR DESCRIPTION
**Pull request by deminy**
This is to remove error control operators (the at sign) in class [\Swoole\FastCGI\HttpResponse](https://github.com/swoole/library/blob/master/src/core/FastCGI/HttpResponse.php).

### CODE REVIEW

File: src/core/FastCGI/HttpResponse.php [hunk 1]
```diff
@@ -37,14 +37,21 @@ public function __construct(array $records = [])
         if (strlen($body) === 0) {
             return;
         }
-        [$headers, $body] = @explode("\r\n\r\n", $body, 2);
+        if (strpos($body, "\r\n\r\n") === false) {
```
Comment (twose) [inline src/core/FastCGI/HttpResponse.php:40]: 这里这么写strpos和explode会扫描两次，不如`$tmp = expload(...)`，然后判断数组长度

File: src/core/FastCGI/HttpResponse.php [hunk 2]
```diff
@@ -37,14 +37,21 @@ public function __construct(array $records = [])
...
+            return;
+        }
+        [$headers, $body] = explode("\r\n\r\n", $body, 2);
         $headers = explode("\r\n", $headers);
         foreach ($headers as $header) {
-            [$name, $value] = @explode(':', $header, 2);
+            if (strpos($header, ':') === false) {
+                continue; // Invalid HTTP header? Ignore it!
+            }
+            [$name, $value] = explode(':', $header, 2);
```
Comment (twose) [inline src/core/FastCGI/HttpResponse.php:50]: 同上

File: src/core/FastCGI/HttpResponse.php [hunk 3]
```diff
@@ -37,14 +37,21 @@ public function __construct(array $records = [])
...
             $name = trim($name);
             $value = trim($value);
             if (strcasecmp($name, 'Status') === 0) {
-                [$statusCode, $reasonPhrase] = @explode(' ', $value, 2);
+                [$statusCode, $reasonPhrase] = explode(' ', $value, 2);
```
Comment (twose) [inline src/core/FastCGI/HttpResponse.php:54]: 如果数组长度为1会产生Notice

File: src/core/FastCGI/HttpResponse.php [hunk 4]
```diff
@@ -37,21 +37,24 @@ public function __construct(array $records = [])
         if (strlen($body) === 0) {
             return;
         }
-        if (strpos($body, "\r\n\r\n") === false) {
+        $array = explode("\r\n\r\n", $body, 2); // An array that contains the HTTP headers and the body.
+        if (count($array) != 2) {
             $this->withStatusCode(502)->withReasonPhrase('Invalid FastCGI Response')->withError($body);
```
Comment (twose) [inline src/core/FastCGI/HttpResponse.php:42]: 最好使用常量, 在`Swoole\Http\Status`

File: src/core/FastCGI/HttpResponse.php [hunk 5]
```diff
@@ -37,21 +37,24 @@ public function __construct(array $records = [])
...
-            [$name, $value] = explode(':', $header, 2);
-            $name = trim($name);
-            $value = trim($value);
+            $name = trim($array[0]);
+            $value = trim($array[1]);
             if (strcasecmp($name, 'Status') === 0) {
-                [$statusCode, $reasonPhrase] = explode(' ', $value, 2);
+                $array = explode(' ', $value, 2); // An array that contains the status code (and the reason phrase).
+                $statusCode = $array[0];
+                $reasonPhrase = $array[1] ?? null;
```
Comment (twose) [inline src/core/FastCGI/HttpResponse.php:57]: 是否有必要给一个默认的reasonPhrase, 在`Swoole\Http\Status`下有方法获取

Comment (deminy) [reply src/core/FastCGI/HttpResponse.php:57]: Default reason phrase is populated at line 65; it works fine as shown in test [\Swoole\FastCGI\HttpResponseTest::testStatus()](https://github.com/swoole/library/blob/error-handling/tests/unit/FastCGI/HttpResponseTest.php#L158).

**Pull request merged**